### PR TITLE
Alacritty version 0.14.0-rc1

### PR DESCRIPTION
**Pull request by chrisduerr**

### CODE REVIEW

File: alacritty_terminal/CHANGELOG.md
Comment (chrisduerr) [inline alacritty_terminal/CHANGELOG.md:1]: Noticed that we completely forgot about this changelog, so I did my best to pull all the delta from the last release. We'll have to make sure we don't forget pulling this back into master.

**Review by kchibisov — APPROVED** [review]
I'd assume that you don't want to deal with unicode-width width yet?

**Comment by chrisduerr** [comment]
> I'd assume that you don't want to deal with unicode-width crate yet?

I forgot about this. But getting that sorted plus ctrl+shift+o would further delay the RC. I'll try to get a new unicode crate written today and otherwise I'd just merge this and leave that for an RC2 maybe?

**Comment by chrisduerr** [comment]
Rebased with all the latest changes applied.

**Pull request merged**